atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweepThread.java

```diff
@@ -213,7 +215,11 @@ private SweepOutcome grabLocksAndRun(SingleLockService locks) throws Interrupted
     }
 
     private long getBackoffTimeWhenSweepHasNotRun() {
-        return 20 * (1000 + sweepPauseMillis.get());
+        return 20 * (1000 + sweepPauseMillis.get()); // 2 minutes by default
```

Comment:
I've always been slightly confused by this - is this only used for the pause before we start sweep? If so, it might be better to defined the backoff time you're adding completely separately from the definition here, unless it should always be 5x this one.

Reply:
The sweep pause millis is the pause between each iteration of sweep, we have a small pause (5 seconds by default) to prevent us hammering the database and we have increased this when a particular DB was under load or decreased it when we knew we had a lot of work to do.

The maths here is to ensure that we always pause for some amount of time when there's a problem in the hope it will clear.

This being 20x the sweep pause, and the new delay being 5x are kind of arbitrary but give sensible values for the range we expect for the sweep pause.  I can add further configuration values or just hardcode the new sleep to 10 minutes.

Reply:
I think hardcoding makes sense - the two don't seem linked to each other. We shouldn't need configuration for this. Legacy sweep has always been quite async and the sweep priority scans should be cheap enough that they are not a problem (or we should fix them further).